lib/eldap/src/eldap.erl

```diff
@@ -217,6 +249,11 @@ modify_password(Handle, Dn, NewPasswd) -> return_value() | {ok, GenPasswd}
 Modify the password of a user. See `modify_password/4`.
 """.
 -doc(#{since => <<"OTP 18.0">>}).
+-spec modify_password(Handle, Dn, NewPasswd) -> ok | {error, term()} | {ok, GenPasswd} when
```

Comment:
According to the old doc this function can also return "{ok, {referral, referrals()}}".

Reply:
fixed